### PR TITLE
feat/capitalize-events

### PR DESCRIPTION
**Pull request by mosheavni**
I have custom resources: kibanas and elasticsearches
they both have `green`, `yellow` or `red` on their status.
not capitalized.
I figured we better capitalize anyway to get the colors right.
before:
image

after:
image

### CODE REVIEW

File: lua/kubectl/utils/events.lua [hunk 1]
```diff
@@ -86,6 +88,7 @@ function M.ColorStatus(status)
     Normal = true,
     VolumeResizeSuccessful = true,
     FileSystemResizeSuccessful = true,
+    SecretSynced = true,
```
Comment (mosheavni) [inline lua/kubectl/utils/events.lua:91]:
```suggestion
```

File: lua/kubectl/utils/events.lua [hunk 2]
```diff
@@ -95,17 +98,22 @@ function M.ColorStatus(status)
     -- Custom statuses
     Active = true,
     True = true,
+    SuccessfullyReconciled = true,
```
Comment (mosheavni) [inline lua/kubectl/utils/events.lua:101]:
```suggestion
    SecretSynced = true,
    SuccessfullyReconciled = true,
```

**Comment by Ramilito** [comment]
There is a capitalization method in the utils/string.lua, do we do something different here?

**Comment by mosheavni** [comment]
didn't know about it. now it's using it

**Comment by Ramilito** [comment]
Love it!

**Comment by Ramilito** [comment]
> didn't know about it. now it's using it

Yeah figured but my git diff on phone is sometimes hard to read 😅

**Pull request merged**